website/fiddle/package.json

```diff
@@ -22,15 +22,15 @@
     "dev": "node patch.js && node build_patch_ui5.js && webpack-dev-server  --content-base public"
   },
   "dependencies": {
-    "@luigi-project/client": "1.22.0",
-    "@luigi-project/core": "1.22.0",
-    "@luigi-project/plugin-auth-oauth2": "1.22.0",
-    "@luigi-project/plugin-auth-oidc": "1.22.0",
-    "@sap-theming/theming-base-content": "~11.1.35",
+    "@luigi-project/client": "1.21.0",
+    "@luigi-project/core": "1.21.0",
+    "@luigi-project/plugin-auth-oauth2": "1.21.0",
+    "@luigi-project/plugin-auth-oidc": "1.21.0",
+    "@sap-theming/theming-base-content": "~11.1.28",
```

Comment:
I can assume this was changed by mistake. please revert it. Then run npm install to revert the package-lock conflict